package.json

```diff
@@ -51,12 +51,12 @@
   "dependencies": {
     "@polymer/iron-resizable-behavior": "^3.0.1",
     "@polymer/polymer": "^3.0.0",
-    "@vaadin/vaadin-combo-box": "^5.0.9",
-    "@vaadin/vaadin-control-state-mixin": "^2.1.3",
-    "@vaadin/vaadin-lumo-styles": "^1.5.0",
-    "@vaadin/vaadin-material-styles": "^1.2.3",
-    "@vaadin/vaadin-text-field": "^2.4.8",
-    "@vaadin/vaadin-themable-mixin": "^1.4.4"
+    "@vaadin/vaadin-combo-box": "6.0.1",
+    "@vaadin/vaadin-control-state-mixin": "2.1.4",
+    "@vaadin/vaadin-lumo-styles": "1.6.1",
+    "@vaadin/vaadin-material-styles": "1.3.2",
+    "@vaadin/vaadin-text-field": "3.0.2",
+    "@vaadin/vaadin-themable-mixin": "1.6.2"
```

Comment:
You should not be requiring exact versions, prefix everything with `^`.

Reply:
Indeed I forgot to allow newer minor & patch versions after testing specific ones for our case.